complex/cargo/list/src/lib.rs

```diff
@@ -204,6 +207,23 @@ impl ListManager {
         }
     }
 
+    pub fn fetch_item_uuids(&self) -> Vec<*const c_char> {
```

Comment:
I am not a huge fan of c types being used inside `ListManager`. `string_to_c_char` calls unsafe code and unsafe code should be kept to the FFI boundary as much as possible which allows this function to be called from elsewhere in Rust without safety worries.

Reply:
Ah, good call! I'll revise this.